editor/code_editor.h

```diff
@@ -38,6 +38,7 @@
 #include "scene/gui/dialogs.h"
 #include "scene/gui/label.h"
 #include "scene/gui/line_edit.h"
+#include "scene/gui/menu_button.h"
```

Comment:
This should be forward declared and included in cpp instead.
(same applies to most of the includes here, but they are out of scope of this PR)